BMM.Core/Translation/en/main.json

```diff
@@ -178,7 +178,11 @@
     "Title": "Favorites",
     "DownloadedContent": "Downloaded Content",
     "FollowedPodcasts": "Followed Podcasts",
-    "MyTracks": "My Tracks"
+    "MyTracks": "My Tracks",
+    "MyPlaylists": "My playlists",
+    "SharedWithMe": "Shared with me",
+    "TrackCountFormat": "{0} Tracks",
```

Comment:
Do you mean that i should introduce Plural And Singular form of this format or that I need to use DocumentsViewModel.PluralTracks entry instead of creating new one (there's difference between them in letter size, but  i'm not sure of we care about it in this case)?

Reply:
I think it should be the same entry.
Also if you look at the design it's always using lowercase.